2996_reflection/reflection.md

```diff
@@ -8391,18 +8393,27 @@ To [cpp.predefined]{.sref}:
 
 ::: std
 ```diff
-  __cpp_impl_coroutine 201902L
-  __cpp_impl_destroying_delete 201806L
-  __cpp_impl_three_way_comparison 201907L
-+ __cpp_impl_reflection 2025XXL
+ [...]
+  __cpp_impl_raw_strings            200710L
+  __cpp_impl_ref_qualifiers         200710L
++ __cpp_impl_reflection             2025XXL
+  __cpp_impl_return_type_deduction  201304L
+  __cpp_impl_rvalue_references      200610L
+ [...]
```

Comment:
This change is unnecessary, don't need more context.

Reply:
Removed the ones around it.

Reply:
I mean, just revert back to what it was before. Having like `impl_coroutine` there was useful to demonstrate that we do this stuff. I just said we don't need _more_ context (not _any_).